v2/thirdparty/pre-built-ui/multitenant-login.mdx

```diff
@@ -0,0 +1,130 @@
+---
+id: multitenant-login
+title: "Multitenant login"
+hide_title: true
+---
+
+import AppInfoForm from "/src/components/appInfoForm"
+import BackendSDKTabs from "/src/components/tabs/BackendSDKTabs";
+import GoFrameworkSubTabs from "/src/components/tabs/GoFrameworkSubTabs";
+import TabItem from '@theme/TabItem';
+import MultitenancyIntro from '../reusableMD/multitenancy/intro.mdx'
+import CreateConfigureTenants from '../reusableMD/multitenancy/create-configure-tenants.mdx'
+import BackendCORS from '../reusableMD/multitenancy/backend-cors.mdx'
+
+:::important
+TODO BANNER This is a paid feature
+
+If you wish to use this feature for free, follow this implementation guide TODO link
```

Comment:
- The TODO link should be under multi tenancy section
- There should be text that tells users how to start a subscription (TBD) - but add that as a thing here..
- Extract this into it's own component that can be imported and used in all pages. Call that component "MultiTenancyPaidBanner"

Reply:
Done